app/models/api/EligibilitySubmissionData.scala

```diff
@@ -17,17 +17,38 @@
 package models.api
 
 import play.api.libs.functional.syntax._
-import play.api.libs.json.{Format, Json, Reads}
+import play.api.libs.json._
+import uk.gov.hmrc.http.InternalServerException
 
 case class EligibilitySubmissionData(threshold: Threshold,
+                                     exceptionOrExemption: String,
                                      estimates: TurnoverEstimates,
                                      customerStatus: CustomerStatus)
 
 object EligibilitySubmissionData {
 
+  val submissionFormat: Format[EligibilitySubmissionData] = (
+    (__ \ "subscription" \ "reasonForSubscription").format[Threshold](Threshold.submissionFormat) and
+    (__ \ "subscription" \ "reasonForSubscription" \ "exemptionOrException").format[String] and
+    (__ \ "subscription" \ "yourTurnover" \ "turnoverNext12Months").format[TurnoverEstimates](TurnoverEstimates.submissionFormat) and
+    (__ \ "admin" \ "additionalInformation" \ "customerStatus").format[CustomerStatus](CustomerStatus.format)
+    ) (EligibilitySubmissionData.apply, unlift(EligibilitySubmissionData.unapply))
+
   val eligibilityReads: Reads[EligibilitySubmissionData] = Reads { json =>
     (
       json.validate[Threshold](Threshold.eligibilityDataJsonReads) and
+      (
+        (json \ "vatRegistrationException").validateOpt[Boolean] and
+        (json \ "vatExemption").validateOpt[Boolean]
+      ) ((exception, exemption) =>
+        (exception.contains(true), exemption.contains(true)) match {
+          case (true, false) => "2"
```

Comment:
Would be good if the values were constants so we know what they're referring to without looking at the EPIDD

Reply:
Also, though a bit more verbose it would be easier to read if it was something like

`(exception.contains(true), exemption.contains(true)) match {
    case (excepted, exempt) if !excepted && !exempt => RegReasons.NoReason
    case (excepted, _) if excepted => RegReasons.Excepted`
    case (_, exempt) if exempt => RegReasons.Exempt`
    case _ => throw ...
}